doc/config.example.yaml

```diff
@@ -195,31 +198,7 @@ dns:
 ################################################################################
 logging:
   root:
```

Comment:
even after changing root to 
` - name: root`
agora still returned parsing errors, so removed the whole block

Reply:
> agora still returned parsing errors, so removed the whole block

Well we really can't merge that. The syntax can be seen in the infrastructure repo, or just by reading the code.
The change is simple: Instead of using:
```yaml
- name: agora.foo
   attribute: value
```

We should use:
```yaml
agora.foo:
  attribute: value
```